types/webpack/index.d.ts

```diff
@@ -264,7 +264,7 @@ declare namespace webpack {
          *
          * @see alias
          */
-        aliasFields?: string[];
+        aliasFields?: string[] | string[][];
```

Comment:
Modified this because I've noticed the unused `CommonArrayOfStringOrStringArrayValues`.